packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -299,6 +300,7 @@ class ThemeData with Diagnosticable {
     TabBarTheme? tabBarTheme,
     TextButtonThemeData? textButtonTheme,
     TextSelectionThemeData? textSelectionTheme,
+    DatePickerThemeData? datePickerTheme,
```

Comment:
These have been sorted since this PR was created. Can you move the data picker entries up under the 'D' components alphabetically?

Reply:
Done👍